src/components/events-list/event-summary.js

```diff
@@ -1,44 +1,40 @@
+import { Table } from '@equinor/eds-core-react';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import RelativeToNow from '../time/relative-to-now';
-import WarningState from './warning-state';
 import EventType from './event-type';
-
+import WarningState from './warning-state';
+import RelativeToNow from '../time/relative-to-now';
 import eventModel from '../../models/event';
-import { isWarningEvent } from '../../utils/event-model';
-import { Table } from '@equinor/eds-core-react';
 
-const EventSummary = ({ event }) => {
-  return (
-    <>
-      <Table.Cell>
-        <RelativeToNow time={event.lastTimestamp} titlePrefix="Start" />
-      </Table.Cell>
-      <Table.Cell>
-        <EventType event={event}></EventType>
-      </Table.Cell>
-      <Table.Cell>
-        {event.involvedObjectKind}/{event.involvedObjectName}
-      </Table.Cell>
-      <Table.Cell className="wrap">
-        <span>
-          {event.reason} - {event.message}
-          {event.involvedObjectState &&
-            event.involvedObjectState.pod &&
-            event.involvedObjectState.pod.restartCount > 0 && (
-              <span>
-                . Restarted {event.involvedObjectState.pod.restartCount} times
-              </span>
-            )}
-        </span>
-      </Table.Cell>
-      <Table.Cell>
-        {isWarningEvent(event) && <WarningState event={event}></WarningState>}
-      </Table.Cell>
-    </>
-  );
-};
+const EventSummary = ({ event }) => (
+  <>
+    <Table.Cell>
+      <RelativeToNow
+        time={event.lastTimestamp}
+        titlePrefix="Start"
+        capitalize
+      />
+    </Table.Cell>
+    <Table.Cell>
+      <EventType event={event}></EventType>
```

Comment:
EventType is only used here and is quite a basic component. Perhaps we should just use a StatusBadge directly? 
```suggestion
      <StatusBadge type={isWarningEvent(event) ? 'warning' : 'success'}>
        {event?.type}
      </StatusBadge>
```

Reply:
Changed to using StatusBadge directly